### DISCUSSION TITLE
q-form rendering is broken but i see no warning or css change with inspection tools

### DISCUSSION THREAD

**Discussion by efotopoulou** [quasarframework/quasar#16583] [General - Components / Directives / etc]
Hi all,
I would really appreciate your feedback in a UI issue.
This is my sign in form.
The first time i visit my app it seems fine and works well.
![image](https://github.com/quasarframework/quasar/assets/1914256/8b95619c-9f13-4ec1-99aa-098c6e611d1d)
After a while the form is still operative but looks weird. The email rules and password Rules are shown on top of the edit field. I can only add my credentials when i click on the label (not the space line itself) and when i write my input (email), my text is overlapped with the rule message.
![image](https://github.com/quasarframework/quasar/assets/1914256/d30b42e0-6a9f-4fce-a90b-7568da889816)
![image](https://github.com/quasarframework/quasar/assets/1914256/120300a7-a13c-423d-be15-203a6b971c77)

I check out the css before(the first time of the page loading) and after of the q-form components and is exactly the same. 
Any idea of what is happening?
It is a very small because the form is still operative...but gives a very bad user experience and i have many complaints from the end users... :-(

**Comment by pdanpdan** [comment]
your repro: https://gitlab.com/netmode/emosociograms

most probably you should check your browser's extensions

**Reply by efotopoulou** [reply]
Do you refer to a specific package in the package.json file?
This happened after a merge request in the project. I got the same error even with the old package.json (the one that this UI issue was not present).

**Reply by metalsadman** [reply]
check the final lock file, others might be using a different version on install when you have transitive dependency numbers in your package.json. also what dan suggested above.
